pallav/fashionTech/pallavBanka/parkingAssignment/Parking.java

```diff
@@ -1,2 +1,173 @@
-package com.fashionTech.pallavBanka.parkingAssignment;
+package fashionTech.pallavBanka.parkingAssignment;
 
+import java.util.Date;
+import java.util.Timer;
+
+abstract class parking{
+    void parkVehicle(vehicle v){
```

Comment:
it should be void parkVehicle(Vehicle v)  -> class name starts with capital letter